configs/PicoFightingBoard/env.ini

```diff
@@ -0,0 +1,5 @@
+[env:pico-fighting-board]
```

Comment:
This is a comment for the image `PinMapping.png`:

Please update from `GP2040-CE Wiring Diagram` to something about `Pico Fighting Board`